mhs/mhs/outbound/request/synchronous/handler.py

```diff
@@ -28,15 +28,16 @@ def initialize(self, workflow: sync_async_workflow.SyncAsyncWorkflow,
         self.async_response_received = tornado.locks.Event()
         self.async_timeout = async_timeout
 
-    async def post(self):
+    async def post(self, interaction_name):
         logging.debug("Client POST received: %s", self.request)
```

Comment:
(Optional as maybe it's worth refactoring it all at a later date) I know you didn't write this but we maybe we should try to update old logging statements as we go?

Reply:
I agree (and @gareth-allan mentioned this to me as a general point as well).